TRACING-4559: clarify replication factor (#82367)
openshift/openshift-docs: modules/distr-tracing-tempo-config-default.adoc

<5> Contains all of the configuration parameters of the TempoStack instance. When a common definition for all Tempo components is required, define it in the `spec` section. When the definition relates to an individual component, place it in the `spec.template.<component>` section.
<6> Storage is specified at instance deployment. See the installation page for information about storage options for the instance.
<7> Defines the compute resources for the Tempo container.

> <8> Configuration for the replication factor. The replication factor is a configuration setting that determines how many ingesters need to acknowledge the data from the distributors before accepting a span.

Reviewer: ```suggestion
<8> Configuration for the replication factor. The replication factor is a configuration setting that determines how many ingesters must acknowledge the data from the distributors before accepting a span.
```

> <8> Configuration for the replication factor. The replication factor is a configuration setting that determines how many copies of your trace data are stored across different instances of Tempo's backend storage. For example, a replication factor of 3 means each trace is written to three different nodes, providing resilience against up to two node failures.

Reviewer 2: We also need to change the upstream API docs https://github.com/grafana/tempo-operator/blob/main/docs/spec/tempo.grafana.com_tempostacks.yaml#L57 as the information is misleading.
